libs/feature/editor/src/lib/components/record-form/form-field/index.ts

```diff
@@ -9,3 +9,4 @@ export * from './form-field-object/form-field-object.component'
 export * from './form-field-array/form-field-array.component'
 export * from './form-field-spatial-extent/form-field-spatial-extent.component'
 export * from './form-field.component'
+export * from './form-field-constraints-shortcuts/form-field-constraints-shortcuts.component'
```

Comment:
Shouldn't it be
```suggestion
export * from './form-field-constraints-shortcuts/constraints.utils'
```
to use the const mentioned above?

Reply:
Oh okay, will try